slides/k8s/volumes.md

```diff
@@ -0,0 +1,184 @@
+# Volumes
+
+- Volumes are special directories that are mounted in containers
+
+- Volumes can have many different purposes:
+
+  - share files and directories between containers running on the same machine
+
+  - share files and directories between containers and their host
+
+  - centralize configuration information in Kubernetes and expose it to containers
+
+  - manage credentials and secrets and expose them securely to containers
+
+  - store persistent data for stateful services
+
+  - access storage systems (like Ceph, EBS, NFS, portworx, and many others)
+
+---
+
+## Kubernetes volumes vs. Docker volumes
+
+- Kubernetes and Docker volumes are very similar
+
+  (the [Kubernetes documentation](https://kubernetes.io/docs/concepts/storage/volumes/) says otherwise ...
+  <br/>
+  but it refers to Docker 1.7, which was released in 2015!)
+
+- Docker volumes allow to share data between containers running on the same host
+
+- Kubernetes volumes allow to share data between containers in the same pod
```

Comment:
> Kubernetes volumes allow to share data between containers in the same pod

Kubernetes volumes allow _us_ to share data between containers in the same pod

Reply:
Fixing too